src/webgl/shaders/cubeFragment.glsl

```diff
@@ -0,0 +1,23 @@
+#version 330 core
```

Comment:
Right now this shader will only work in WebGL 2 mode. For shaders we want to work in both versions, we add these macros to the start https://github.com/processing/p5.js/blob/main/src/webgl/shaders/webgl2Compatibility.glsl by concatenating it here:
https://github.com/processing/p5.js/blob/bda63ff6de3b96d6b51e262221a578dea194759e/src/webgl/p5.RendererGL.js#L83-L85
...and by adding an appropriate prefix for the vertex/fragment shaders like this:
https://github.com/processing/p5.js/blob/bda63ff6de3b96d6b51e262221a578dea194759e/src/webgl/p5.RendererGL.js#L1745-L1748

To make this file use it, it would involve taking out the `#version` (that's added by the prefix we add), and replacing `out` and `in` with the macros `OUT` and `IN` which get `#define`d to the appropriate keyword depending on what WebGL version is being used. It also means not explicitly declaring `out vec4 FragColor` and instead using `OUT_COLOR`, which we also automatically add a definition for.